flink/v1.20/flink-runtime/LICENSE

```diff
@@ -464,47 +425,92 @@ License text:
 
 --------------------------------------------------------------------------------
 
-This binary artifact includes Project Nessie with the following in its NOTICE
-file:
+This binary artifact contains Project Nessie.
 
-| Dremio
-| Copyright 2015-2017 Dremio Corporation
-|
-| This product includes software developed at
-| The Apache Software Foundation (http://www.apache.org/).
+Copyright: Copyright 2015-2017 Dremio Corporation
+Home page: https://projectnessie.org/
+License: http://www.apache.org/licenses/LICENSE-2.0
 
 --------------------------------------------------------------------------------
 
-This binary includes code from Apache Commons.
+This binary artifact contains Apache HttpComponents Client.
 
-* Core ArrayUtil.
+Copyright: 1999-2022 The Apache Software Foundation.
```

Comment:
As discussed, I'd remove these based on [the comment here](https://github.com/apache/iceberg-python/commit/fbdf04b0f4d83719b26facbac25b4695ff8f0629#commitcomment-140697012).

Reply:
Agree. Let me do some cleanup 😄

Reply:
I did some cleaning on `LICENSE` and `NOTICE`.